generators/go/internal/testdata/sdk/environments-core/fixtures/core/core.go

```diff
@@ -22,6 +23,16 @@ type HTTPClient interface {
 	Do(*http.Request) (*http.Response, error)
 }
 
+// EncodeURL encodes the given arguments into the URL, escaping
+// values as needed.
+func EncodeURL(urlFormat string, args ...interface{}) string {
```

Comment:
Could this potentially be a helper function given it's re-used many places?

Reply:
It is! This is actually included in every generated SDK's `core` package, which is why we see it in all these different `seed` and `internal/testdata` snapshots.

Reply:
🤦‍♂️totally missed that, my bad!